gems/pending/util/MiqSockUtil.rb

```diff
@@ -8,6 +8,9 @@ def self.getHostName
   # Return the cannonical name for host in DNS
   def self.getFullyQualifiedDomainName
     Socket.gethostbyname(Socket.gethostname).first
+  rescue
+    $log.error "The appliance hostname could not be resolved. Please make sure to set a valid fully qualified hostname."
```

Comment:
@chessbyte I think the BZ error happens when the hostname is blank? @mcornea is that true?